_maps/templates/shelter_2.dmm

```diff
@@ -8,6 +8,17 @@
 	},
 /turf/closed/wall/mineral/titanium/survival/pod,
 /area/survivalpod)
+"c" = (
```

Comment:
What's going on here ?

Reply:
I changed the curtains icons since their icons are different now